docs/DeveloperGuide.md

```diff
@@ -312,16 +312,79 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorTrack`, the **Actor** is the `Tutor` and the **Person** is the `Tutee`, unless specified otherwise)
+
+**Use case: View all Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+
+    Use case ends.
+
+**Extensions**
+* 2a. The list is empty.
+* *a. Connection lost.
+    * *a1. System informs the user.
+
+  Use case ends.
+
+**Use case: Add a Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+3.  User requests to add a specific person in the list
+4.  System adds the person
+
+    Use case ends.
+
+**Extensions**
+* 3a. The given format is invalid.
+    * 3a1. System shows an error message.
+
+      Use case resumes at step 2.
+* *a. Connection lost.
+    * *a1. System informs the user.
+
+      Use case ends.
+
+**Use case: Update a Person**
+
+**MSS**
+
+1.  User requests to list all persons
+2.  System shows a list of all persons
+3.  User requests to update a specific person in the list
+4.  System update the person
+
+    Use case ends.
+
+**Extensions**
+* 2a. The list is empty.
+* 3a. The given format is invalid.
+    * 3a1. System shows an error message.
+
+      Use case resumes at step 2.
+* 3b. The given index is invalid.
+    * 3a1. System shows an error message.
+
+      Use case resumes at step 2.
+* *a. Connection lost.
+    * *a1. System informs the user.
+
+      Use case ends.
 
 **Use case: Delete a person**
```

Comment:
If you want, this was the one I wrote for delete use case which includes a confirmation to delete.

**Use case: UC02 - Delete a student**

**MSS**

1.  Tutor requests to list students.
2.  TutorTrack displays the list of students.
3.  Tutor requests to delete a specific patient in the list
4.  TutorTrack requests for confirmation
5.  Tutor confirms
6.  TutorTrack deletes the student from the database.
    Use case ends.

**Extensions**

* 2a. List is empty.

    * 2a1. TutorTrack shows an error message.

    Use case ends.

* 3a. The given index is invalid.

    * 3a1. TutorTrack shows an error message.
    * 3a2. TutorTrack requests for the correct data
    * 3a3. Tutor enters new data
  
    Steps 3a2-3a3 are repeated until the data entered are correct.
  
    Use case resumes from step 4.

* 4a. Tutor cancels deletion.

    * 4a1. TutorTrack displays confirmation message

* 4b. Tutor enteres invalid syntax for confirmation.

    * 4a1. TutorTrack displays error message
    * 4a2. TutorTrack requests for the correct data
    * 4a3. Tutor enters new data

  Steps 4a2-4a3 are repeated until the data entered is correct.
  Use case continues from step 5.